vaadin-testbench-unit-shared/src/main/java/com/vaadin/flow/component/grid/GridTester.java

```diff
@@ -560,8 +560,7 @@ private String getValueProviderString(int row, Grid.Column targetColumn)
             ColumnPathRenderer renderer = (ColumnPathRenderer) targetColumn
                     .getRenderer();
 
-            Field f = ColumnPathRenderer.class
-                    .getDeclaredField("provider");
+            Field f = ColumnPathRenderer.class.getDeclaredField("provider");
```

Comment:
I would revert this formatting change, so that the pr only focus on the shortcut issue

Reply:
👍

Reply:
I just pushed mvn formatter:format. I can revert it.